WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -1017,3 +1018,13 @@ extension OrderDetailsDataSource {
         static let paidByCustomerCell = 1
     }
 }
+
+// MARK: - Private Utils
+
+private extension OrderDetailsDataSource {
+    enum Localization {
+        static func pluralizedProducts(count: Int) -> String {
+            count > 1 ? Title.products : Title.product
```

Comment:
Non-blocking 💬  It'd be nice if we could use a `.stringsdict` for noun plurals ([Apple doc](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/LocalizingYourApp/LocalizingYourApp.html) > "Handling Noun Plurals and Units of Measurement"), though it seems like it's not supported yet from paaHJt-c3-p2. Do you think it's worth a ping to Platform 9 since the last conversation was in 2018?

Reply:
Yeah, I looked into this not too long ago when I wrote [this WPiOS documentation](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/docs/localization.md). Documentation is awesome because I wrote in there something I forgot: 

> GlotPress currently does not support pluralization using the .stringsdict file. So, right now, you have to support plurals manually by having separate localized strings.

The limitation is in GlotPress. 😐 

Based on the discussion in https://github.com/wordpress-mobile/WordPress-iOS/issues/6327, it looks like it's still not resolved on the GlotPress side. 

Once that is resolved, I think we also need to move out of `localize.py` to something that parses the `.stringsdict` too.